src/networks.json

```diff
@@ -1349,6 +1349,20 @@
     "start": 3369285,
     "logo": "ipfs://QmXgGmxDAW2Mheum3WX7Q52Zi9hvE17Zp1pVtZbcVdThh4"
   },
+  "239": {
+    "key": "239",
+    "name": "AITD Testnet",
+    "chainId": 239,
+    "network": "Testnet",
+    "testnet": true,
+    "multicall": "0x71f63b35f4a90835678fcb3b53207d7b2ae252fa",
```

Comment:
Can you verify the contract code here http://aitd-explorer-pre.aitdcoin.com/addr/0x71f63b35f4a90835678fcb3b53207d7b2ae252fa#tab_addr_3

Reply:
The testnet browser has no verification function. 
I will directly submit the mainnet related data later.